frontend/src/app/data-entry/agent-form/agent-identification-form/agent-identification-form.component.html

```diff
@@ -36,11 +36,12 @@
         <label class="form-label" for="input-isgroup">
             Is this agent a person or a group?
         </label>
-        <select id="input-isgroup" class="form-select"
+        <select id="input-isgroup" class="form-select" [class.is-invalid]="form.invalid"
             [formControl]="form.controls.isGroup">
             <option [ngValue]="false">Person</option>
             <option [ngValue]="true">Group</option>
         </select>
+        <p class="invalid-feedback">A single-person agent cannot be linked to more than one historical person.</p>
```

Comment:
This is checking whether the form as a whole is invalid, so the message will also show up when there is an issue with the name:

![screenshot of form, showing an empty name field, where the message about groups is also displayed](https://github.com/user-attachments/assets/979454fa-f882-4762-8f29-e59ffd27870f)

Reply:
Right, good spot!